### ISSUE TITLE
Add labels to forms for accessibility: 0.5

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#271]
> Labels or instructions must be provided when content requires user input. Forms on the NYU ISAW’s are missing associated labels. The radio buttons are missing an associated label and groups of radio buttons with the same name attribute must be contained within a fieldset element with appropriate legend value.

![screen shot 2018-04-27 at 3 22 06 pm](https://user-images.githubusercontent.com/263285/39383409-c5b9f000-4a2e-11e8-996b-978c88ec56ec.png)

> Screen readers often switch to “Forms Mode” when they are processing content within a ```<form>``` element. In this mode they usually only read aloud form elements such as ```<input>, <select>, <textarea>, <legend>, and <label>```. It is critical to include form instructions in a way that can be read aloud.

WCAG 2.0 Compliance: [3.2.2 Labels or Instructions](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-cues.html)

> Labels or instructions are provided when content requires user input. (Level A)

> Resource: https://www.w3.org/WAI/tutorials/forms/instructions/

Traceability: [Microassist Report 1 > WCAG Compliance > Form Fields and Labels](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.xw038tyjrwgn)

**Comment by paregorios** [comment]
See also #242

**Comment by paregorios** [comment]
Yeah, I think that is probably it. Closing this as complete (noting that #242 remains in workflow=deploy).

**Comment by paregorios** [comment]
@alecpm it looks like I failed to check to see that we'd grouped the radio buttons in a ```fieldset``` element as specified at the top of this ticket, but now I see on production that we didn't do that. Should I open a new ticket for that aspect, or should we address here?

**Comment by alecpm** [comment]
Sorry I missed that bit. I've got a little time left from the last iteration so I should be able to do that on this ticket.

**Comment by paregorios** [comment]
OK. Should I move this ticket back to "doing" in the project board?

**Comment by alecpm** [comment]
Yes, I think that’s the way to go.

**Comment by paregorios** [comment]
done

**Comment by alecpm** [comment]
I've implemented the fieldset + legend on staging. It required some styling and markup changes to not look completely strange. Let me know if you'd like to adjust styling or change the legend.

**Comment by paregorios** [comment]
Maybe just a few style tweaks:

```css
#search-events { border: 1px solid #231f20 }
#search-events > form { margin: 1em }
#search-events > form fieldset { padding: 0px }
#search-events > form fieldset > legend { padding: 0px }
#search-events > form fieldset > legend::after { content: ":" }
```

**and**

remove the ```#search-events form fieldset { border 1px solid #231f20; }``` directive

The idea is to get:

![screen shot 2018-10-08 at 6 04 48 pm](https://user-images.githubusercontent.com/263285/46638061-b32afc00-cb24-11e8-90c4-1aa952101bb5.png)

instead of 

![screen shot 2018-10-08 at 6 00 36 pm](https://user-images.githubusercontent.com/263285/46637950-18cab880-cb24-11e8-93f0-925e7ff3ebd1.png)

**Comment by paregorios** [comment]
@skleinfeldt what should my expectations be about finishing off this issue? I realize I was slow to respond on review.

**Comment by alecpm** [comment]
I should be able to get this done in a short amount of time later this week. Along with finishing #242

**Comment by alecpm** [comment]
The updated styling is up on staging.

**Comment by paregorios** [comment]
Looks good, please merge to master and deploy to production.

**Comment by alecpm** [comment]
This is deployed, but seeing it on the live server I think we need one more styling tweak to add some margin between the event search form and the view more button.

**Comment by paregorios** [comment]
@alecpm agreed. Maybe something like:

```css
#search-events { margin-top: 2em }
```

?

**Comment by alecpm** [comment]
Done.

**Comment by paregorios** [comment]
confirmed deployed
